src/swap/defi/0x/0xGasless.ts

```diff
@@ -27,6 +27,9 @@ const swapInfo: EdgeSwapInfo = {
   supportEmail: 'support@edge.app'
 }
 
+const EDGE_FEE_PERCENTAGE = 0.0075
```

Comment:
this should be have an override that can come from env.json on the GUI. See how we do Thorchain